src/components/dashboard/Sidebar/Sidebar.js

```diff
@@ -27,11 +28,11 @@ const Sidebar = () => {
       name: 'Trash',
       icon: <FaTrash className="Sidebar__icon" />,
     },
-    {
-      path: '/profile',
-      name: 'Profile',
-      icon: <FaUserCircle className="Sidebar__icon" />,
-    },
+    // {
+    //   path: '/profile',
+    //   name: 'Profile',
+    //   icon: <FaUserCircle className="Sidebar__icon" />,
+    // },
```

Comment:
delete comments

Reply:
Commented part contains link that'll be added after current submission hence added the commented part